tldr/daemon/startup.py

```
@@ -51,14 +51,88 @@ def _get_socket_path(project: Path) -> Path:
+    if not lock_acquired:
+        return
+
+    # Lock is free - check if it's a zombie
+    if pid_path.exists():
+        try:
+            pid = int(pid_path.read_text().strip())
+            if not is_process_alive(pid):
+                # Zombie! Remove both files
+                lock_path.unlink(missing_ok=True)
+                pid_path.unlink(missing_ok=True)
+                logger.info(f"Cleaned up zombie lock for project {project}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)